avocado/utils/nvme.py

```
@@ -26,7 +26,9 @@
 import logging
 import os
 import re
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)